tailwind.config.js

```diff
@@ -4,6 +4,28 @@ module.exports = {
     './components/**/*.{js,ts,jsx,tsx,md}',
   ],
   darkMode: "class",
+  safelist: [
```

Comment:
Rather than adding elements to the safelist here we should check for other alternatives like making sure we are adding styles without using string concatenation to create class names:

https://v2.tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html